### PR TITLE
[CORE-220]: add bot blocked event + event handling

### PR DESCRIPTION
**Pull request by Anglopol**

### CODE REVIEW

**Review by zhelezkov — APPROVED** [review]
lgtm

File: src/dialect/dialect-events.ts [hunk 1]
```diff
@@ -16,3 +16,13 @@ export interface ThreadMessageSentEvent extends ThreadEvent {
   recipients: PublicKey[];
   message: string;
 }
+
+export enum TelegramEventType {
```
Comment (tsmbl) [inline src/dialect/dialect-events.ts:20]: I think this event should be placed somewhere in telegram package, wdyt?

File: src/telegram/telegram-bot.ts [hunk 2]
```diff
@@ -109,6 +109,7 @@ export class TelegramBot {
       this.logger.error(
```
Comment (tsmbl) [inline src/telegram/telegram-bot.ts:109]: I think we should rework logging, since we don't expect error logs when telegram bot is blocked by user

**Review by tsmbl — CHANGES_REQUESTED** [review]
Overall good job, left a couple of comments

**Review by tsmbl — APPROVED** [review]
lgtm, nice job

**Pull request merged**